Makefile

```diff
@@ -7,13 +7,13 @@ include versioning.mk
 
 DEV_ENV_IMAGE := quay.io/deis/go-dev:0.9.0
 DEV_ENV_WORK_DIR := /go/src/github.com/deis/${SHORT_NAME}
-DEV_ENV_CMD := docker run --rm -e CGO_ENABLED=0 -v ${PWD}:${DEV_ENV_WORK_DIR} -w ${DEV_ENV_WORK_DIR} ${DEV_ENV_IMAGE}
+DEV_ENV_CMD := docker run --rm -e CGO_ENABLED=0 -v ${CURDIR}:${DEV_ENV_WORK_DIR} -w ${DEV_ENV_WORK_DIR} ${DEV_ENV_IMAGE}
 
 # Common flags passed into Go's linker.
 LDFLAGS := "-s -X main.version=${VERSION}"
 
 # Docker Root FS
-BINDIR := ./rootfs/bin
+BINDIR := ${CURDIR}/rootfs/bin
```

Comment:
@jackfrancis I think this will cause some problems when used for the `go build` command inside the container. for example, this is the `go build` command that's run (taken from [build logs](https://travis-ci.org/deis/workflow-manager/builds/115182553#L244)): 

```console
go build -o /home/travis/build/deis/workflow-manager/rootfs/bin/boot -a -installsuffix cgo -ldflags "-s -X main.version=git-b04ff6d" boot.go
```

Note that the `-o` should point to `rootfs/bin/boot`. maybe just change the `-o ${BINDIR}/boot` to `-o rootfs/bin/boot` in the `go build` command below?